versions/development/SPEC.md

```diff
@@ -254,20 +254,18 @@ These are common among many of the following sections
 ```
 $ws = (0x20 | 0x9 | 0xD | 0xA)+
 $identifier = [a-zA-Z][a-zA-Z0-9_]+
-$string = "([^\\\"\n]|\\[\\"\'nrbtfav\?]|\\[0-7]{1,3}|\\x[0-9a-fA-F]+|\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*"
-$string = '([^\\\'\n]|\\[\\"\'nrbtfav\?]|\\[0-7]{1,3}|\\x[0-9a-fA-F]+|\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*'
 $boolean = 'true' | 'false'
 $integer = [1-9][0-9]*|0[xX][0-9a-fA-F]+|0[0-7]*
 $float = (([0-9]+)?\.([0-9]+)|[0-9]+\.|[0-9]+)([eE][-+]?[0-9]+)?
 ```
 
 `$string` can accept the following between single or double-quotes:
 
-* Any character not in set: `\\`, `"` (or `'` for single-quoted string), `\n`
-* An escape sequence starting with `\\`, followed by one of the following characters: `\\`, `"`, `'`, `[nrbtfav]`, `?`
-* An escape sequence starting with `\\`, followed by 1 to 3 digits of value 0 through 7 inclusive.  This specifies an octal escape code.
-* An escape sequence starting with `\\x`, followed by hexadecimal characters `0-9a-fA-F`.  This specifies a hexadecimal escape code.
-* An escape sequence starting with `\\u` or `\\U` followed by either 4 or 8 hexadecimal characters `0-9a-fA-F`.  This specifies a unicode code point
+* Any character not in set: `\`, `"` (or `'` for single-quoted string), `\n`
+* An escape sequence starting with `\`, followed by one of the following characters: `\nrbtfav?`
```

Comment:
Can we break this into separate sub-bullets indicating what each escape sequence is interpreted as? 

... and we should probably double check that all of these actually make sense...

Reply:
these are the same as the c89 escape sequences. Are we happy to refer to other documents or do we want to define here?

Reply:
I'd be happy to reference out to other docs or redefine. One reason for listing out is to make it explicit which ones we're actually supporting since some of these do seem unnecessary (eg IIRC `b` is "backspace", which IMO we could probably do without)